### DISCUSSION TITLE
Feature request: sorting of repositories

### DISCUSSION THREAD

**Discussion by xhrnca00** [community/community#48465] [Mobile]
<details>
<summary>It would be easier to find stuff if it was possible to sort repositories</summary>
GitHub for iOS, Version 1.100.0 (91333308)
Device: iPhone 8 (iOS 16.3)
</details>

Hey, I use GitHub mobile quite a lot. I've noticed, that my repositories are sorted in "some" way. I usually have to scroll about ten repositories to find what I'm looking for, meanwhile the second spot in the list is a project that was finished a year ago and will never get any commits.
I think it would be useful to have the option to sort repositories by most recent commit date, most commits, open PRs, …
I am sure this would help users find the repository they are looking for sooner.
Thanks for reading!

**Comment by candyho** [comment]
Hey thanks for your feedback! I will add this to our internal repo to track the request.

**Comment by marcotrosi** [comment]
I found this discussion because I was searching for "how to set the default repo sorting for an organization" on Github.
The sorting feature has been added some time ago, but it seems it was forgotten to implement a configurable default value for sorting in the organization settings of Github.
would be nice to get a section in the organization configuration where such default values can be configured.

also this discussion here goes in the same direction
https://github.com/orgs/community/discussions/50697

**Comment by ivanperez-keera** [comment]
@candyho Can you also please allow sorting by name (alphabetically)? This really helps especially when working with many repos.

**Comment by gregorybchris** [comment]
Given the standard desktop experience is default sorted chronologically, you'd think mobile could paginate chronologically as well. Any chance this has been implemented and it's hidden behind some setting?
